src/lib/hooks/useCeramicSDK.js

```diff
@@ -0,0 +1,21 @@
+import { useEffect, useState } from 'react'
+import { CeramicSDK } from '@gooddollar/ceramic-seed-sdk'
+import Web3 from 'web3'
+const sdk = new CeramicSDK('https://ceramic-clay.3boxlabs.com')
+
+const useCeramicSDK = () => {
+  const [initialized, setInitialized] = useState(false)
+  const [ceramicSdk, setCeramicSdk] = useState({})
+  useEffect(() => {
+    const privateAccountKey = JSON.parse(localStorage.getItem('GD_masterSeed'))
```

Comment:
have you verified that this effect doesnt run again whenever ConnectedAccounts re-renders?
you  might  need to use a ref here

Reply:
yes verified that, because use effect has empty dependency array it runs only 1 time on mount